### PR TITLE
Fix formatting and add ascp password env variable

### PR DESCRIPTION
**Pull request by ElDeveloper**
There were a few problems with the new configuration that we got after
contacting EBI, in short:
    - Passwords had to be updated, so we've centralized the retrieval of
      the credentials to be only in this file for the subprocesses that
      need them.
    - There was a missin `"` in the formatting of a string to generate
      the CURL commands.

---

Please note that once this branch is merged into master and deployed, we will have to modify the profile file for the system to be in concordance with this change set.

### CODE REVIEW

File: python_code/live_ebi_sra_rest_services.py [hunk 1]
```diff
@@ -104,7 +104,8 @@ def generate_curl_command(self):
         curl_command = (
             'curl -F "SUBMISSION=@{0}" -F "STUDY=@{1}" -F "SAMPLE=@{2}" '
             '-F "RUN=@{3}" -F"EXPERIMENT=@{4}" '
-            '"https://www.ebi.ac.uk/ena/submit/drop-box/submit/{5}'
+            '             ' # legacy white-spaces
```
Comment (adamrp) [inline python_code/live_ebi_sra_rest_services.py]: Do we actually need these...?  haha

**Comment by adamrp** [comment]
Looks good to me!  I think we should test without those legacy whitespaces to see if everything works and, if it does, we should remove them

File: python_code/live_ebi_sra_rest_services.py (hunk 1 above)
Comment (ElDeveloper) [inline python_code/live_ebi_sra_rest_services.py]: Not really, no. I'm pretty sure that these can be removed, but wanted to be consistent with what's on live right now. If we decide to remove them, we must add a space at the beginning of the next line though.

**Comment by ElDeveloper** [comment]
Ok, that sounds reasonable. I'm going to assume they do not matter ...

**Comment by ElDeveloper** [comment]
@teravest @adamrp Do we want to keep holding off on this one? I don't remember why were we holding off though :shipit:

**Comment by teravest** [comment]
This looks good to me what about the legacy whitespaces? how did that test go?

**Comment by ElDeveloper** [comment]
I don't think we need the spaces and it should work without them but I left them there because that was in the code before. Will remove them if you guys think I should.

**Comment by teravest** [comment]
If it works without them we should remove them.

**Comment by adamrp** [comment]
agree with @teravest

**Comment by teravest** [comment]
@ElDeveloper  did you test this without the spaces?

**Comment by ElDeveloper** [comment]
Sorry, I completely forgot about this pull request, all should be fixed now.

**Pull request merged**